Pipfile

```diff
@@ -18,6 +18,7 @@ celery = "*"
 django-celery-results = "*"
 redis = "*"
 "discord.py" = "*"
+pre-commit = "*"
 
 [dev-packages]
 pre-commit = "*"
```

Comment:
Isn't this already in the dev-packages section below? I think if you run `pipenv install --dev` it should be installed.

Reply:
oh you're right, I forgot the --dev, thanks!